node/src/constants/amplitude.rs

```diff
@@ -0,0 +1,58 @@
+use runtime_common::{Balance, UNIT};
+use spacewalk_primitives::{oracle::Key, Asset, CurrencyId, CurrencyId::XCM, VaultCurrencyPair};
+
+pub const AMPLITUDE_PARACHAIN_ID: u32 = 2124;
+pub const FOUCOCO_PARACHAIN_ID: u32 = 2124;
+pub const AMPLITUDE_INITIAL_ISSUANCE: Balance = 200_000_000 * UNIT;
+pub const INITIAL_ISSUANCE_PER_SIGNATORY: Balance = 200 * UNIT;
+pub const INITIAL_COLLATOR_STAKING: Balance = 10_010 * UNIT;
+
+pub const OFF_CHAIN_WORKER_ADDRESS: &str = "6m69vWMouLarYCbJGJisVaDDpfNGETkD5hsDWf2T7osW4Cn1";
+
+pub const TOKEN_DECIMALS: u32 = 12;
+
+pub const INITIAL_AMPLITUDE_SUDO_SIGNATORIES: [&str; 5] = [
+	"6nJwMD3gk36fe6pMRL2UpbwAEjDdjjxdngQGShe753pyAvCT",
+	"6i4xDEE1Q2Bv8tnJtgD4jse4YTAzzCwCJVUehRQ93hCqKp8f",
+	"6n62KZWvmZHgeyEXTvQFmoHRMqjKfFWvQVsApkePekuNfek5",
+	"6kwxQBRKMadrY9Lq3K8gZXkw1UkjacpqYhcqvX3AqmN9DofF",
+	"6kKHwcpCVC18KepwvdMSME8Q7ZTNr1RoRUrFDH9AdAmhL3Pt",
+];
+
+pub const INITIAL_AMPLITUDE_VALIDATORS: [&str; 8] = [
```

Comment:
The list of invulnerables (collators) for Foucoco needs to be different. It is currently defined to be:
```
"6ihktBwyFJYjE1LKdqoAWzo5VDPJJGso9D5iASZyhuN5JvGH",
"6mbXa9Qca6B6cX51cbtfWWLhup84rMoMFCxNHjso15GBFyGh",
"6mMdv2wmb4Cp8PAtDLF1WTh1wLPwPbETwtcjqgJLskdB8EYo",
"6kL1dzcBJiLgMdAT1qDFD79CLupX1gCCF8RSg5Dh5qRgQeCx"
```

Reply:
Changed @TorstenStueber. I created a new constant called `INITIAL_FOUCOCO_VALIDATORS` which is used to create the invulnerables (collators) variable in the foucoco config.